metagraph/src/common/sorted_sets/sorted_set_disk_base.cpp

```diff
@@ -207,7 +204,7 @@ void SortedSetDiskBase<T>::merge_l1(const std::string &chunk_file_prefix,
     merge_files(to_merge, on_new_item);
     encoder.finish();
 
-    (*l1_chunk_count)++;
+    *l1_chunk_count += 1;
```

Comment:
any particular reason why? don't these two use the same memory order anyway?

Reply:
I thought this looks simpler, no other reasons